cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +90,66 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
+	counter := 0
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe != nil && cut.LivenessProbe.Exec != nil {
+				counter++
+				if cut.LivenessProbe.PeriodSeconds > minExecProbePeriod {
+					tnf.ClaimFilePrintf("Container %s is using exec probes, PeriodSeconds of LivenessProbe: %s", cut,
+						cut.LivenessProbe.PeriodSeconds)
+					compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Liveness exec probe has a PeriodSeconds greater than 10", true))
+				} else {
+					tnf.ClaimFilePrintf("Container %s is not using of exec probes, PeriodSeconds of LivenessProbe: %s", cut,
+						cut.LivenessProbe.PeriodSeconds)
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Ensure limited use of exec probes", false))
+				}
+			}
+			if cut.StartupProbe != nil && cut.StartupProbe.Exec != nil {
+				counter++
+				if cut.StartupProbe.PeriodSeconds > minExecProbePeriod {
+					tnf.ClaimFilePrintf("Container %s is using exec probes, PeriodSeconds of StartupProbe: %s", cut,
+						cut.StartupProbe.PeriodSeconds)
+					compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "StartupProbe exec probe has a PeriodSeconds greater than 10", true))
+				} else {
+					tnf.ClaimFilePrintf("Container %s is not using of exec probes, PeriodSeconds of StartupProbe: %s", cut,
+						cut.StartupProbe.PeriodSeconds)
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Ensure limited use of exec probes", false))
+				}
+			}
+			if cut.ReadinessProbe != nil && cut.ReadinessProbe.Exec != nil {
+				counter++
+				if cut.ReadinessProbe.PeriodSeconds > minExecProbePeriod {
+					tnf.ClaimFilePrintf("Container %s is using exec probes, PeriodSeconds of ReadinessProbe: %s", cut,
+						cut.ReadinessProbe.PeriodSeconds)
+					compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "ReadinessProbe exec probe has a PeriodSeconds greater than 10", true))
+				} else {
+					tnf.ClaimFilePrintf("Container %s is not using of exec probes, PeriodSeconds of ReadinessProbe: %s", cut,
+						cut.ReadinessProbe.PeriodSeconds)
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Ensure limited use of exec probes", false))
+				}
+			}
+		}
+	}
+
+	// If there >=10 exec probes, mark the entire cluster as a failure
+	if counter >= maxNumberOfExecProbes {
+		tnf.ClaimFilePrintf("CNF has 10 or more exec probes")
+		nonCompliantObjects = append(nonCompliantObjects, testhelper.NewReportObject("CNF has 10 or more exec probes", testhelper.CnfType, false).
```

Comment:
"CNF has 10 or more exec probes" -> fmt.Sprintf("CNF has %d exec probes",counter)

Reply:
Also update the report objects like: 
nonCompliantObjects = append(nonCompliantObjects, testhelper.NewReportObject(fmt.Sprintf("CNF has 10 or more exec probes (%d exec probes)",counter), testhelper.CnfType, false).